shared/api/storage_pool_volume.go

```diff
@@ -31,6 +31,7 @@ type StorageVolume struct {
 	Name             string   `json:"name" yaml:"name"`
 	Type             string   `json:"type" yaml:"type"`
 	UsedBy           []string `json:"used_by" yaml:"used_by"`
+	Node             string   `json:"node" yaml:"node"`
```

Comment:
Should be "Location" instead and should be under an "API extension" comment too.

Reply:
Done